client/debug/legacybech32pubkey/bech32pubkey.go

```diff
@@ -0,0 +1,85 @@
+package bech32pubkey
```

Comment:
It seems to me this whole file is a copy/paste of `types/bech32/legacybech32` with some naming changes, is that right?

Can we use that package instead and remove this file?

Reply:
@amaurym  Done!